client/v2/client.go

```diff
@@ -18,7 +20,10 @@ import (
 // UDPPayloadSize is a reasonable default payload size for UDP packets that
 // could be travelling over the internet.
 const (
-	UDPPayloadSize = 512
+	UDPPayloadSize = 1500
+
+	newlineString = "\n"
+	commaString   = ","
```

Comment:
Remove these constants. They add an extra level of indirection. They don't change between platforms: `\n` vs `\r\n` for text on unix-like OSes vs. Windows.

As a Go style note: avoid naming variables with they're type. The `String` suffix would be unnecessary in this case, even if the constants made sense to keep.